components/style/values/specified/image.rs

```diff
@@ -98,6 +99,9 @@ impl Parse for Image {
         if let Ok(gradient) = input.try(|i| Gradient::parse(context, i)) {
             return Ok(GenericImage::Gradient(gradient));
         }
+        if let Ok(paint_worklet) = input.try(|i| PaintWorklet::parse(context, i)) {
```

Comment:
Err, that sounded quite inquisitive, I meant "You're going to need to remove" :)